library/include/borealis/view.hpp

```diff
@@ -101,6 +101,8 @@ class View
 
     bool hidden = false;
 
+    bool clipping = true;
```

Comment:
would it not be "clipped" instead of "clipping"? or "clippingEnabled"?

Reply:
good point!